certbot/certbot/_internal/client.py

```diff
@@ -251,9 +251,9 @@ def perform_registration(acme: acme_client.ClientV2, config: configuration.Names
         newreg = messages.NewRegistration.from_data(
             email=config.email, external_account_binding=eab)
 
-        if tos_cb and "terms_of_service" in acme.directory.meta and \
-            acme.directory.meta.terms_of_service:
-            tos_cb(acme.directory.meta.terms_of_service)
+        tos = acme.directory.meta.terms_of_service
```

Comment:
Why were we checking in the first place if this can't even error, smh

(this is rhetorical, I see the test code)